starters/nextjs/basic/src/app/ssg/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { getDateString, getRandomUUID } from "../utils";
+
+export default function Page() {
+  return (
+    <main className="content">
+      <h1 className="heading">Sample title for SSG</h1>
```

Comment:
Should this be updated to something else?

Reply:
good catch, thanks, kept it as just SSG